2013-09-28 19:49:40 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [1045] Access denied for user \'root\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /
Stack Trace:
#0 C:\wamp\www\lib\Cake\Model\Datasource\DboSource.php(262): Mysql->connect()
#1 C:\wamp\www\lib\Cake\Model\ConnectionManager.php(107): DboSource->__construct(Array)
#2 C:\wamp\www\lib\Cake\Model\Model.php(3221): ConnectionManager::getDataSource('default')
#3 C:\wamp\www\lib\Cake\Model\Model.php(1107): Model->setDataSource('default')
#4 C:\wamp\www\lib\Cake\Model\Model.php(3243): Model->setSource('users')
#5 C:\wamp\www\lib\Cake\Model\Model.php(1318): Model->getDataSource()
#6 C:\wamp\www\lib\Cake\View\Helper\FormHelper.php(199): Model->schema()
#7 C:\wamp\www\lib\Cake\View\Helper\FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 C:\wamp\www\app\View\Elements\form_login.ctp(20): FormHelper->create('User', Array)
#9 C:\wamp\www\lib\Cake\View\View.php(945): include('C:\wamp\www\app...')
#10 C:\wamp\www\lib\Cake\View\View.php(907): View->_evaluate('C:\wamp\www\app...', Array)
#11 C:\wamp\www\lib\Cake\View\View.php(1208): View->_render('C:\wamp\www\app...', Array)
#12 C:\wamp\www\lib\Cake\View\View.php(414): View->_renderElement('C:\wamp\www\app...', Array, Array)
#13 C:\wamp\www\app\View\Users\login.ctp(2): View->element('form_login')
#14 C:\wamp\www\lib\Cake\View\View.php(945): include('C:\wamp\www\app...')
#15 C:\wamp\www\lib\Cake\View\View.php(907): View->_evaluate('C:\wamp\www\app...', Array)
#16 C:\wamp\www\lib\Cake\View\View.php(471): View->_render('C:\wamp\www\app...')
#17 C:\wamp\www\lib\Cake\Controller\Controller.php(948): View->render(NULL, NULL)
#18 C:\wamp\www\lib\Cake\Routing\Dispatcher.php(194): Controller->render()
#19 C:\wamp\www\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 C:\wamp\www\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 C:\wamp\www\index.php(43): require('C:\wamp\www\app...')
#22 {main}
2013-09-28 19:50:01 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [1045] Access denied for user \'root\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /
Stack Trace:
#0 C:\wamp\www\lib\Cake\Model\Datasource\DboSource.php(262): Mysql->connect()
#1 C:\wamp\www\lib\Cake\Model\ConnectionManager.php(107): DboSource->__construct(Array)
#2 C:\wamp\www\lib\Cake\Model\Model.php(3221): ConnectionManager::getDataSource('default')
#3 C:\wamp\www\lib\Cake\Model\Model.php(1107): Model->setDataSource('default')
#4 C:\wamp\www\lib\Cake\Model\Model.php(3243): Model->setSource('users')
#5 C:\wamp\www\lib\Cake\Model\Model.php(1318): Model->getDataSource()
#6 C:\wamp\www\lib\Cake\View\Helper\FormHelper.php(199): Model->schema()
#7 C:\wamp\www\lib\Cake\View\Helper\FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 C:\wamp\www\app\View\Elements\form_login.ctp(20): FormHelper->create('User', Array)
#9 C:\wamp\www\lib\Cake\View\View.php(945): include('C:\wamp\www\app...')
#10 C:\wamp\www\lib\Cake\View\View.php(907): View->_evaluate('C:\wamp\www\app...', Array)
#11 C:\wamp\www\lib\Cake\View\View.php(1208): View->_render('C:\wamp\www\app...', Array)
#12 C:\wamp\www\lib\Cake\View\View.php(414): View->_renderElement('C:\wamp\www\app...', Array, Array)
#13 C:\wamp\www\app\View\Users\login.ctp(2): View->element('form_login')
#14 C:\wamp\www\lib\Cake\View\View.php(945): include('C:\wamp\www\app...')
#15 C:\wamp\www\lib\Cake\View\View.php(907): View->_evaluate('C:\wamp\www\app...', Array)
#16 C:\wamp\www\lib\Cake\View\View.php(471): View->_render('C:\wamp\www\app...')
#17 C:\wamp\www\lib\Cake\Controller\Controller.php(948): View->render(NULL, NULL)
#18 C:\wamp\www\lib\Cake\Routing\Dispatcher.php(194): Controller->render()
#19 C:\wamp\www\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 C:\wamp\www\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 C:\wamp\www\index.php(43): require('C:\wamp\www\app...')
#22 {main}
2013-09-28 20:20:09 Error: [MissingControllerException] Controller class LoginController could not be found.
Exception Attributes: array (
  'class' => 'LoginController',
  'plugin' => NULL,
)
Request URL: /login
Stack Trace:
#0 C:\wamp\www\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-28 21:08:34 Error: [MissingControllerException] Controller class HomeController could not be found.
Exception Attributes: array (
  'class' => 'HomeController',
  'plugin' => NULL,
)
Request URL: /home/gerenciador
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-28 21:08:53 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Photo.order' in 'order clause'
Request URL: /gerenciador
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Datasource\DboSource.php(460): PDOStatement->execute(Array)
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Datasource\DboSource.php(426): DboSource->_execute('SELECT `Photo`....', Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Datasource\DboSource.php(666): DboSource->execute('SELECT `Photo`....', Array, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Datasource\DboSource.php(1077): DboSource->fetchAll('SELECT `Photo`....', false)
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(2696): DboSource->read(Object(Photo), Array)
#5 C:\Users\Lucas\Documents\GitHub\desafio\app\Plugin\Gerenciador\Controller\PhotosController.php(10): Model->find('all', Array)
#6 [internal function]: PhotosController->index()
#7 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(486): ReflectionMethod->invokeArgs(Object(PhotosController), Array)
#8 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#9 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(PhotosController), Object(CakeRequest), Object(CakeResponse))
#10 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2013-09-28 21:11:42 Error: [MissingTableException] Table videos for model Video was not found in datasource default.
Exception Attributes: array (
  'table' => 'videos',
  'class' => 'Video',
  'ds' => 'default',
)
Request URL: /gerenciador/videos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3243): Model->setSource('videos')
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(2696): Model->getDataSource()
#2 C:\Users\Lucas\Documents\GitHub\desafio\app\Plugin\Gerenciador\Controller\VideosController.php(6): Model->find('all')
#3 [internal function]: VideosController->index()
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(486): ReflectionMethod->invokeArgs(Object(VideosController), Array)
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(VideosController), Object(CakeRequest), Object(CakeResponse))
#7 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2013-09-28 21:12:08 Error: [MissingTableException] Table videos for model Video was not found in datasource default.
Exception Attributes: array (
  'table' => 'videos',
  'class' => 'Video',
  'ds' => 'default',
)
Request URL: /gerenciador/videos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3243): Model->setSource('videos')
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(2696): Model->getDataSource()
#2 C:\Users\Lucas\Documents\GitHub\desafio\app\Plugin\Gerenciador\Controller\VideosController.php(6): Model->find('all')
#3 [internal function]: VideosController->index()
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(486): ReflectionMethod->invokeArgs(Object(VideosController), Array)
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(VideosController), Object(CakeRequest), Object(CakeResponse))
#7 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2013-09-28 21:12:10 Error: [MissingTableException] Table videos for model Video was not found in datasource default.
Exception Attributes: array (
  'table' => 'videos',
  'class' => 'Video',
  'ds' => 'default',
)
Request URL: /gerenciador/videos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3243): Model->setSource('videos')
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(2696): Model->getDataSource()
#2 C:\Users\Lucas\Documents\GitHub\desafio\app\Plugin\Gerenciador\Controller\VideosController.php(6): Model->find('all')
#3 [internal function]: VideosController->index()
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(486): ReflectionMethod->invokeArgs(Object(VideosController), Array)
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(VideosController), Object(CakeRequest), Object(CakeResponse))
#7 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2013-09-28 21:16:48 Error: [SocketException] Invalid email: "from@your_app_domain.com"
Request URL: /users/remember_password
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Network\Email\CakeEmail.php(570): CakeEmail->_setEmail('_from', 'from@your_app_d...', NULL)
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Network\Email\CakeEmail.php(356): CakeEmail->_setEmailSingle('_from', 'from@your_app_d...', NULL, 'From requires o...')
#2 C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\UsersController.php(256): CakeEmail->from('from@your_app_d...')
#3 [internal function]: UsersController->remember_password()
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#7 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2013-09-28 22:59:47 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'root\'@\'localhost\' (using password: NO)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:00:32 Error: [MissingControllerException] Controller class GerenciarController could not be found.
Exception Attributes: array (
  'class' => 'GerenciarController',
  'plugin' => NULL,
)
Request URL: /desafio/gerenciar
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-28 23:00:43 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'root\'@\'localhost\' (using password: NO)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:16:27 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'root\'@\'localhost\' (using password: NO)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:17:59 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'root\'@\'localhost\' (using password: NO)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:18:01 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'root\'@\'localhost\' (using password: NO)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:23:00 Error: [MissingConnectionException] Database connection "Postgres" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Postgres',
  'message' => 'SQLSTATE[08006] [7] could not translate host name "http://marcos.web.gp.utfpr.edu.br" to address: Name or service not known',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Postgres->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:24:47 Error: [MissingConnectionException] Database connection "Postgres" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Postgres',
  'message' => 'SQLSTATE[08006] [7] could not translate host name "http://marcos.web.gp.utfpr.edu.br" to address: Name or service not known',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Postgres->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:25:43 Error: [MissingConnectionException] Database connection "Postgres" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Postgres',
  'message' => 'SQLSTATE[08006] [7] server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Postgres->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:32:41 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:34:07 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:34:08 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:34:09 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:35:19 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-28 23:35:24 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[28000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /desafio/
Stack Trace:
#0 /var/www/desafio/lib/Cake/Model/Datasource/DboSource.php(262): Mysql->connect()
#1 /var/www/desafio/lib/Cake/Model/ConnectionManager.php(107): DboSource->__construct(Array)
#2 /var/www/desafio/lib/Cake/Model/Model.php(3221): ConnectionManager::getDataSource('default')
#3 /var/www/desafio/lib/Cake/Model/Model.php(1107): Model->setDataSource('default')
#4 /var/www/desafio/lib/Cake/Model/Model.php(3243): Model->setSource('users')
#5 /var/www/desafio/lib/Cake/Model/Model.php(1318): Model->getDataSource()
#6 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(199): Model->schema()
#7 /var/www/desafio/lib/Cake/View/Helper/FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 /var/www/desafio/app/View/Elements/form_login.ctp(20): FormHelper->create('User', Array)
#9 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#10 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#11 /var/www/desafio/lib/Cake/View/View.php(1208): View->_render('/var/www/desafi...', Array)
#12 /var/www/desafio/lib/Cake/View/View.php(414): View->_renderElement('/var/www/desafi...', Array, Array)
#13 /var/www/desafio/app/View/Users/login.ctp(2): View->element('form_login')
#14 /var/www/desafio/lib/Cake/View/View.php(945): include('/var/www/desafi...')
#15 /var/www/desafio/lib/Cake/View/View.php(907): View->_evaluate('/var/www/desafi...', Array)
#16 /var/www/desafio/lib/Cake/View/View.php(471): View->_render('/var/www/desafi...')
#17 /var/www/desafio/lib/Cake/Controller/Controller.php(948): View->render(NULL, NULL)
#18 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(194): Controller->render()
#19 /var/www/desafio/lib/Cake/Routing/Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-29 05:04:03 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Datasource\DboSource.php(262): Mysql->connect()
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\ConnectionManager.php(107): DboSource->__construct(Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3221): ConnectionManager::getDataSource('default')
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(1107): Model->setDataSource('default')
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3243): Model->setSource('users')
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(1318): Model->getDataSource()
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\Helper\FormHelper.php(199): Model->schema()
#7 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\Helper\FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 C:\Users\Lucas\Documents\GitHub\desafio\app\View\Elements\form_login.ctp(20): FormHelper->create('User', Array)
#9 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(945): include('C:\Users\Lucas\...')
#10 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(907): View->_evaluate('C:\Users\Lucas\...', Array)
#11 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(1208): View->_render('C:\Users\Lucas\...', Array)
#12 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(414): View->_renderElement('C:\Users\Lucas\...', Array, Array)
#13 C:\Users\Lucas\Documents\GitHub\desafio\app\View\Users\login.ctp(2): View->element('form_login')
#14 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(945): include('C:\Users\Lucas\...')
#15 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(907): View->_evaluate('C:\Users\Lucas\...', Array)
#16 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(471): View->_render('C:\Users\Lucas\...')
#17 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(948): View->render(NULL, NULL)
#18 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(194): Controller->render()
#19 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-29 05:07:12 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Datasource\DboSource.php(262): Mysql->connect()
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\ConnectionManager.php(107): DboSource->__construct(Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3221): ConnectionManager::getDataSource('default')
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(1107): Model->setDataSource('default')
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3243): Model->setSource('users')
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(1318): Model->getDataSource()
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\Helper\FormHelper.php(199): Model->schema()
#7 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\Helper\FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 C:\Users\Lucas\Documents\GitHub\desafio\app\View\Elements\form_login.ctp(20): FormHelper->create('User', Array)
#9 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(945): include('C:\Users\Lucas\...')
#10 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(907): View->_evaluate('C:\Users\Lucas\...', Array)
#11 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(1208): View->_render('C:\Users\Lucas\...', Array)
#12 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(414): View->_renderElement('C:\Users\Lucas\...', Array, Array)
#13 C:\Users\Lucas\Documents\GitHub\desafio\app\View\Users\login.ctp(2): View->element('form_login')
#14 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(945): include('C:\Users\Lucas\...')
#15 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(907): View->_evaluate('C:\Users\Lucas\...', Array)
#16 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(471): View->_render('C:\Users\Lucas\...')
#17 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(948): View->render(NULL, NULL)
#18 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(194): Controller->render()
#19 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-29 05:08:31 Error: [MissingConnectionException] Database connection "Mysql" is missing, or could not be created.
Exception Attributes: array (
  'class' => 'Mysql',
  'message' => 'SQLSTATE[HY000] [1045] Access denied for user \'desafio\'@\'localhost\' (using password: YES)',
  'enabled' => true,
)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Datasource\DboSource.php(262): Mysql->connect()
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\ConnectionManager.php(107): DboSource->__construct(Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3221): ConnectionManager::getDataSource('default')
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(1107): Model->setDataSource('default')
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3243): Model->setSource('users')
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(1318): Model->getDataSource()
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\Helper\FormHelper.php(199): Model->schema()
#7 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\Helper\FormHelper.php(453): FormHelper->_introspectModel('User', 'fields')
#8 C:\Users\Lucas\Documents\GitHub\desafio\app\View\Elements\form_login.ctp(20): FormHelper->create('User', Array)
#9 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(945): include('C:\Users\Lucas\...')
#10 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(907): View->_evaluate('C:\Users\Lucas\...', Array)
#11 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(1208): View->_render('C:\Users\Lucas\...', Array)
#12 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(414): View->_renderElement('C:\Users\Lucas\...', Array, Array)
#13 C:\Users\Lucas\Documents\GitHub\desafio\app\View\Users\login.ctp(2): View->element('form_login')
#14 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(945): include('C:\Users\Lucas\...')
#15 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(907): View->_evaluate('C:\Users\Lucas\...', Array)
#16 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\View\View.php(471): View->_render('C:\Users\Lucas\...')
#17 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(948): View->render(NULL, NULL)
#18 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(194): Controller->render()
#19 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#20 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#21 {main}
2013-09-29 16:25:17 Error: [MissingControllerException] Controller class Controller could not be found.
Exception Attributes: array (
  'class' => 'Controller',
  'plugin' => NULL,
)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 16:25:21 Error: [MissingControllerException] Controller class Controller could not be found.
Exception Attributes: array (
  'class' => 'Controller',
  'plugin' => NULL,
)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 16:25:24 Error: [MissingControllerException] Controller class Controller could not be found.
Exception Attributes: array (
  'class' => 'Controller',
  'plugin' => NULL,
)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 16:31:19 Error: Fatal Error (4): syntax error, unexpected end of file, expecting function (T_FUNCTION) in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\AppController.php, line 75]
2013-09-29 16:31:19 Error: [FatalErrorException] syntax error, unexpected end of file, expecting function (T_FUNCTION)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 75)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 75, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\Users\Lucas\...', 75, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 16:31:29 Error: Fatal Error (4): syntax error, unexpected end of file, expecting function (T_FUNCTION) in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\AppController.php, line 75]
2013-09-29 16:31:29 Error: [FatalErrorException] syntax error, unexpected end of file, expecting function (T_FUNCTION)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 75)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 75, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\Users\Lucas\...', 75, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 16:31:30 Error: Fatal Error (4): syntax error, unexpected end of file, expecting function (T_FUNCTION) in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\AppController.php, line 75]
2013-09-29 16:31:30 Error: [FatalErrorException] syntax error, unexpected end of file, expecting function (T_FUNCTION)
Request URL: /
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 75)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 75, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\Users\Lucas\...', 75, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 16:32:09 Error: Fatal Error (4): syntax error, unexpected end of file, expecting function (T_FUNCTION) in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\AppController.php, line 79]
2013-09-29 16:32:09 Error: [FatalErrorException] syntax error, unexpected end of file, expecting function (T_FUNCTION)
Request URL: /users/login
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 79)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 79, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\Users\Lucas\...', 79, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 16:32:10 Error: Fatal Error (4): syntax error, unexpected end of file, expecting function (T_FUNCTION) in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\AppController.php, line 79]
2013-09-29 16:32:10 Error: [FatalErrorException] syntax error, unexpected end of file, expecting function (T_FUNCTION)
Request URL: /users/login
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 79)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 79, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\Users\Lucas\...', 79, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 16:33:06 Error: [ForbiddenException] Você não tem permissão para executar esta ação.
Request URL: /
Stack Trace:
#0 [internal function]: UsersController->index()
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(486): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#4 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2013-09-29 16:39:28 Error: [MissingTableException] Table videos for model Video was not found in datasource default.
Exception Attributes: array (
  'table' => 'videos',
  'class' => 'Video',
  'ds' => 'default',
)
Request URL: /gerenciador/videos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(3243): Model->setSource('videos')
#1 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Model\Model.php(2696): Model->getDataSource()
#2 C:\Users\Lucas\Documents\GitHub\desafio\app\Plugin\Gerenciador\Controller\VideosController.php(6): Model->find('all')
#3 [internal function]: VideosController->index()
#4 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Controller\Controller.php(486): ReflectionMethod->invokeArgs(Object(VideosController), Array)
#5 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(187): Controller->invokeAction(Object(CakeRequest))
#6 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Routing\Dispatcher.php(162): Dispatcher->_invoke(Object(VideosController), Object(CakeRequest), Object(CakeResponse))
#7 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2013-09-29 16:51:04 Error: [MissingControllerException] Controller class PhotosController could not be found.
Exception Attributes: array (
  'class' => 'PhotosController',
  'plugin' => NULL,
)
Request URL: /photos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 16:51:44 Error: Fatal Error (1): Class 'GerenciadorAppController' not found in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\PhotosController.php, line 3]
2013-09-29 16:51:44 Error: [FatalErrorException] Class 'GerenciadorAppController' not found
Request URL: /photos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Class 'Gerencia...', 'C:\Users\Lucas\...', 3)
#1 [internal function]: ErrorHandler::handleError(1, 'Class 'Gerencia...', 'C:\Users\Lucas\...', 3, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Class 'Gerencia...', 'C:\Users\Lucas\...', 3, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 16:59:19 Error: Fatal Error (4): syntax error, unexpected 'if' (T_IF) in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\PhotosController.php, line 17]
2013-09-29 16:59:19 Error: [FatalErrorException] syntax error, unexpected 'if' (T_IF)
Request URL: /photos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 17)
#1 [internal function]: ErrorHandler::handleError(4, 'syntax error, u...', 'C:\Users\Lucas\...', 17, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 4, 'syntax error, u...', 'C:\Users\Lucas\...', 17, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 16:59:35 Error: Fatal Error (1): Call to undefined method PhotosController::layout() in [C:\Users\Lucas\Documents\GitHub\desafio\app\Controller\PhotosController.php, line 10]
2013-09-29 16:59:35 Error: [FatalErrorException] Call to undefined method PhotosController::layout()
Request URL: /photos
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Error\ErrorHandler.php(184): ErrorHandler::handleFatalError(1, 'Call to undefin...', 'C:\Users\Lucas\...', 10)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to undefin...', 'C:\Users\Lucas\...', 10, Array)
#2 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(931): call_user_func('ErrorHandler::h...', 1, 'Call to undefin...', 'C:\Users\Lucas\...', 10, Array)
#3 C:\Users\Lucas\Documents\GitHub\desafio\lib\Cake\Core\App.php(904): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-09-29 17:00:03 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:00:48 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:00:50 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:00:57 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:01:02 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:05:46 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:04 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:08 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:31 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:36 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:37 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:43 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:50 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:06:55 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:07:04 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:09:05 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:09:11 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:10:52 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:11:07 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:13:21 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:13:28 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:13:40 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:13:45 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:14:02 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 17:14:11 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 C:\Users\Lucas\Documents\GitHub\desafio\app\webroot\index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 15:46:05 Error: [MissingControllerException] Controller class JsController could not be found.
Exception Attributes: array (
  'class' => 'JsController',
  'plugin' => NULL,
)
Request URL: /js/.lib/modernizr.js
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 19:16:59 Error: [MissingControllerException] Controller class PhpmyController could not be found.
Exception Attributes: array (
  'class' => 'PhpmyController',
  'plugin' => NULL,
)
Request URL: /phpmy
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 19:17:04 Error: [MissingControllerException] Controller class PhpmyadminController could not be found.
Exception Attributes: array (
  'class' => 'PhpmyadminController',
  'plugin' => NULL,
)
Request URL: /phpmyadmin
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 19:18:07 Error: [MissingControllerException] Controller class PhpmyadminController could not be found.
Exception Attributes: array (
  'class' => 'PhpmyadminController',
  'plugin' => NULL,
)
Request URL: /phpmyadmin
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 19:18:09 Error: [MissingControllerException] Controller class PhpmyadminController could not be found.
Exception Attributes: array (
  'class' => 'PhpmyadminController',
  'plugin' => NULL,
)
Request URL: /phpmyadmin
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 19:20:18 Error: [MissingControllerException] Controller class PhpmyadminController could not be found.
Exception Attributes: array (
  'class' => 'PhpmyadminController',
  'plugin' => NULL,
)
Request URL: /phpmyadmin/index.php?token=1aface50c798d64abba24feef26f3909
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 20:18:41 Error: [MissingControllerException] Controller class PhpmyadminController could not be found.
Exception Attributes: array (
  'class' => 'PhpmyadminController',
  'plugin' => NULL,
)
Request URL: /phpmyadmin/db_structure.php?goto=db_sql.php&db=desafio&server=1&token=1aface50c798d64abba24feef26f3909
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-09-29 20:18:45 Error: [MissingControllerException] Controller class PhpmyadminController could not be found.
Exception Attributes: array (
  'class' => 'PhpmyadminController',
  'plugin' => NULL,
)
Request URL: /phpmyadmin/main.php?token=1aface50c798d64abba24feef26f3909
Stack Trace:
#0 /var/www/desafio/app/webroot/index.php(111): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
